src/BookNavigator/bookmarks/ia-bookmarks.js

```diff
@@ -229,17 +229,14 @@ class IABookmarks extends LitElement {
     return bookmark;
   }
 
-  fetchBookmarks() {
-    return this.api.getAll().then(res => res.text()).then((res) => {
-      let response;
-      try {
-        response = JSON.parse(res);
-      } catch (e) {
-        response = { error: e.message };
-      }
-      return response;
-    }).then((response) => {
-      const {
+  async fetchBookmarks() {
```

Comment:
This one looks like it's tackled by @sancodes ' PR, so leaving it there :) Can you revert this file?